coffee_testing/src/lib.rs

```diff
@@ -42,7 +42,7 @@ pub mod macros {
 
     #[macro_export]
     macro_rules! httpd {
-        ($dir:expr, $port:expr, $($opt_args:tt)*) => {
+        ($dir:expr, $port:expr, $cln_path:expr, $($opt_args:tt)*) => {
```

Comment:
no need to had this here, you can add this to the macros string, see next suggestion